client/me/help/help-contact-us-footer.tsx

```diff
@@ -0,0 +1,41 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { Button, CompactCard, Gridicon } from '@automattic/components';
+import { HelpCenter } from '@automattic/data-stores';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
```

Comment:
Nit: there is no other `useDispatch` here, `useDispatch` would work.

```suggestion
import { useDispatch } from '@wordpress/data';
```

Reply:
Right, but I wanted to keep it distinct from "normal" `useDispatch`:
 - To make it clear it's not the one from `react-redux`.
 - If at any point we need to use `useDispatch` from `react-redux` here, it won't require renaming the existing use - smaller change, better for `git blame`, etc.

This is however new code for me, so let me know if I missed something and happy to just use `useDispatch` 🙇‍♂️

Reply:
Yes no problem at all. I just thought it was an artifact of copying other code.

Reply:
> I just thought it was an artifact of copying other code.

In a way, it is ;) I copied this from another file where indeed both `useDispatch`es were used. But I liked the differentiation for the reasons above, so I kept it as-is.